### ISSUE TITLE
3.2.5. Custom element reactions missing diagram img

### ISSUE THREAD

**Issue by scottaohara** [w3c/html#1620]
In section [3.2.5](http://w3c.github.io/html/dom.html#sec-custom-element-reactions) there is text stating "This is all summarised in the following schematic diagram:" but the diagram is missing, and only the alt text is shown.
